CHANGELOG.md

```diff
@@ -5,42 +5,45 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 ## [Unreleased][unreleased]
 
 ### Added 
-- Support 10.3 tokens
-- Add "?ping" for easier testing
-- (DotNet) Support Windows authentication
-- Support federated service
+- Support for 10.3 tokens
+- Added "?ping" for easier testing
+- (DotNet) Support for Windows authentication
+- Support for federated services
 
 ### Fixed
-- Better referrer handling
-- Support ? in password
-- Multiple redirects
-- Handle token expiration
-- Remove HTTP range-request Header
+- Better referer handling
+- Support for '?' in passwords
+- Handling of multiple redirects
+- Handling of token expiration
+
+### Removed
```

Comment:
> I think the change log should reflect a user perspective of functionality
*ahhh* now i think we're onto something substantive.  i'd argue that the changelog should read a lot less like end-user oriented release notes that put a shine on 'whats new' and a lot more like tangible line item details of each and every change/refactor for developers so that they know *exactly* what they're getting into when bumping their own project from one version to the next.

that being said, my recollection of the range-request headers change in particular is hazy at best, so i don't say this to defend my copyedit to the death.